lib/devise.rb

```diff
@@ -286,6 +286,10 @@ module Test
   mattr_accessor :token_generator
   @@token_generator = nil
 
+  # Store devise notification delivery method, default is deliver now
+  mattr_accessor :deliver_later_option
```

Comment:
We should add a new commenting section to the initializer template to document this for new apps.